reviews/2022-cert-manager-annual.md

```diff
@@ -0,0 +1,93 @@
+
+We don't closely track raw numbers of cert-manager users, either on cert-manager's [Artifact Hub package](https://artifacthub.io/packages/helm/cert-manager/cert-manager), visitors to the [cert-manager.io](https://cert-manager.io) or on the cert-manager [quay.io registry](https://quay.io/repository/jetstack/cert-manager-controller).
+
+That said, as an example of scale, the main cert-manager controller container image on quay.io reports roughly 1M-1.5M+ "actions" daily, which likely indicates widespread use and deployment although image cacheing and mirroring may make that an underestimate of true usage. In addition, our best estimates are that the cert-manager website averages around 1.5k unique visitors per day, and that our helm charts see around 1M downloads per day, but these numbers could be out of date and may be undercounts.
+
+As another rough indication of usage, the number of stars on the main cert-manager repository continues to increase steadily and linearly, passing the 8,000 star milestone in [early November 2021](https://certmanager.devstats.cncf.io/d/3/stars-and-forks-by-repository?orgId=1).
+
+## Subprojects
+
+cert-manager itself isn't the only project under the cert-manager umbrella; we've also been increasing the quantity and quality of subprojects and adding more open-source tooling to make it easier to manage Cloud Native certificates and identities.
+
+These subprojects include:
+
+- [istio-csr](https://github.com/cert-manager/istio-csr): Enables the use of cert-manager for issuing [Istio](https://istio.io/) certificates
+- [approver-policy](https://github.com/cert-manager/approver-policy): Adds the `CertificateRequestPolicy` resource to define policy over whether a given certificate should be signed by cert-manager
+- [csi-driver](https://github.com/cert-manager/csi-driver): Kubernetes CSI plugin to facilitate seamlessly requesting and mounting certificate key pairs into pods
+- [csi-lib](https://github.com/cert-manager/csi-lib): A generalised library for building CSI drivers that interact with cert-manager's `CertificateRequest` resource
+- [trust](https://github.com/cert-manager/trust): cert-manager, but for certificate trust bundles rather than for issuing certificates
+
+Importantly, we also received the donation of the [`aws-privateca-issuer`](https://github.com/cert-manager/aws-privateca-issuer) from AWS this year and incorporated this external issuer into the cert-manager organization. We hope to start a trend of companies in the PKI / certificate issuance space working with us to assemble the best possible collection of these addons for end-users to take advantage of.
+
+## Current Goals
```

Comment:
in the review process document it says we should answer the question:

> How has the project performed against its goals since the last review? 

Do we have a prior review with goals, and if so, is that something we can follow up on here?

Reply:
Thanks, good question!

As far as I'm aware, this is the first time there's been a cert-manager annual review (there isn't a previous one in the [upstream list](https://github.com/cncf/toc/tree/main/reviews)) since this is cert-manager's first full year in the CNCF as a sandbox project. As such, there's no "last" review to compare against and I just skipped that question when writing the review.

It might be worth alluding to this being the first annual review more explicitly though; I'll take a stab at that :+1: